core/contracts/Voting.sol

```diff
@@ -187,6 +193,7 @@ contract Voting is Testable, Ownable, OracleInterface, VotingInterface, Encrypte
         votingToken = VotingToken(_votingToken);
         identifierWhitelist = IdentifierWhitelistInterface(_identifierWhitelist);
         finder = Finder(_finder);
+        rewardsExpirationTimeout = 60 * 60 * 24 * 14; // 14 days.
```

Comment:
should we pass this in in construction?

Reply:
Done. I kinda feel like it's a waste though, since we could always set the value to whatever we want right after construction but w/e.